src/components/Project/test/ProjectExecutions.test.tsx

```diff
@@ -20,59 +20,116 @@ import {
 import { failedToLoadExecutionsString } from '../constants';
 import { ProjectExecutions } from '../ProjectExecutions';
 
-jest.mock('components/Executions/Tables/WorkflowExecutionsTable');
+import { APIContext } from 'components/data/apiContext';
+import { mockAPIContextValue } from 'components/data/__mocks__/apiContext';
+import { getUserProfile } from 'models/Common/api';
+import { UserProfile } from 'models/Common/types';
 
-const defaultQueryParams = {
-    [sortQueryKeys.direction]: SortDirection[SortDirection.DESCENDING],
-    [sortQueryKeys.key]: executionSortFields.createdAt
-};
+jest.mock('components/Executions/Tables/WorkflowExecutionsTable');
 
 describe('ProjectExecutions', () => {
     let basicPythonFixture: ReturnType<typeof basicPythonWorkflow.generate>;
     let failedTaskFixture: ReturnType<typeof oneFailedTaskWorkflow.generate>;
-    let executions: Execution[];
+    let executions1: Execution[];
+    let executions2: Execution[];
```

Comment:
Can you give these more descriptive names? It's hard to tell what is different between them or what use case is being tested with each of them.